components/pipeline-service/production/kflux-rhel-p01/deploy.yaml

```diff
@@ -1414,7 +1414,7 @@ spec:
       - args:
         - -pprof-address
         - "6060"
-        image: quay.io/konflux-ci/pipeline-service-exporter:d6d4f669a0818ddf72117207d45a3e664ad7c4cd
+        image: quay.io/konflux-ci/pipeline-service-exporter:9d2439c8a77d2ce0527cc5aea3fc6561b7671b48
```

Comment:
Note: there were a number of these updates applied when I ran `./hack/generate-deploy-config.sh -c components/pipeline-service/production/`... Not sure if they are intentionally different or if the template is simply out of date